packages/functions/src/processor/buffer-reducing.js

```diff
@@ -1,5 +1,4 @@
 import * as Sentry from '@sentry/serverless'
-import { Bucket } from 'sst/node/bucket'
```

Comment:
Sadly we cannot use this binding from new SST version... Given we rely on dynamic bucket names by environment, we need to provide the bucket name as ENV var (like we do in w3infra).